app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::UsersController < ApplicationController
+  def show
+    user = User.find(params['id'])
+    render json: user, serializer: UserShowSerializer
+  rescue StandardError => e
+    render json: { message: e.message }, status: 404
```

Comment:
What is the error message?

Reply:
The error message is a dynamic one we get from rails! When we send in an invalid id, the message is Couldn't find User with 'id'=abc :)